src/main/java/eu/yals/json/EndpointNotFoundJson.java

```diff
@@ -0,0 +1,57 @@
+package eu.yals.json;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.google.gson.annotations.Since;
+import eu.yals.json.internal.Json;
+import kong.unirest.HttpMethod;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * This struct of JSON send when API endpoint is not found
+ *
+ * @since 2.7
+ */
+@Since(2.7)
+public class EndpointNotFoundJson extends Json {
+
+    @Since(2.7)
+    private final String error = "Endpoint not found";
+
+    @Since(2.7)
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    private Endpoint endpoint;
+
+    public static EndpointNotFoundJson createWithEndpoint(@NotNull HttpMethod method, @NotNull String path) {
+        EndpointNotFoundJson self = new EndpointNotFoundJson();
+        self.endpoint = Endpoint.create(method, path);
+        return self;
+    }
+
+    public static EndpointNotFoundJson create() {
+        return new EndpointNotFoundJson();
+    }
+
+    @Since(2.7)
+    public static class Endpoint {
+        @Since(2.7)
+        String method;
+
+        @Since(2.7)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)